scripts/build.py

```diff
@@ -77,11 +77,16 @@
 MANIFEST_FILE_PATH = os.path.join('manifest.json')
 
 REMOVE_WS = re.compile(r'\s{2,}').sub
-YUICOMPRESSOR_DIR = os.path.join(
-    '..', 'oppia_tools', 'yuicompressor-2.4.8', 'yuicompressor-2.4.8.jar')
+
+# This path is only used for java command line. But the Windows style path
```

Comment:
Sorry -- can you elaborate on this? What is a "java command line"?

I am a little confused because I would have thought you'd need to use the Windows filepath convention in the Windows terminal anyway. Is that not the case?

Reply:
So this path is used to run subprocess `java ../oppia_tools/yuicompress...`. If the path is in Windows-style, there will be a syntax error, but it is working correctly with a Unix-style path. 

Windows terminal accepts both Unix-style and Windows-style path (at least for most case). Also, the path is interpreted by the Java program, so we are safe to use Unix one.